host.json

```diff
@@ -15,11 +15,10 @@
   },
   "logging": {
     "logLevel": {
-      "default": "Error",
       "Function.Info": "Error",
-      "Function.BlobEventProcessor": "Information",
-      "Function.ErrorRetryFunction": "Information",
-      "Function.ErrorRetryAllFunction": "Information"
+      "Function.BlobEventProcessor": "Error",
```

Comment:
As [discussed in another project](https://github.com/pagopa/pagopa-fdr-2-event-hub/pull/28#discussion_r2336058343), why not add:
AzureFunctionsJobHost__logging__logLevel__Function__<FUNCTION_NAME>: "<LOG_LEVEL>"
in Helm environment file and make it more customizable? If values in K8s ConfigMap are changed for debug purposes, they can override that value IIRC.
Ditto for the other functions.